onnxruntime/core/session/inference_session.cc

```diff
@@ -244,6 +379,12 @@ common::Status InferenceSession::Load(std::function<common::Status(std::shared_p
     // all steps complete, mark the model as loaded.
     is_model_loaded_ = true;
 
+    // since model load was successful, we don't need to hang on to the member 'model_proto_' anymore
+    // (free up the resource if applicable)
+    if (model_proto_ != nullptr) {
```

Comment:
Do you need the "if"?

Reply:
Can remove the if.  Will leave the comment.